### ISSUE TITLE
Epsilon wave

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7595]
**Preferred term label:**
Epsilon wave

**Synonyms**

**Definition (free text, please give PubMed ID)**
An electric signal of depolarization observed between the end of the QRS complex and the beginning of the T wave. The ε wave is found in the right precordial leads, where the QRS complex is broader than the in the left precordial leads (difference ≥25 ms) in arrhythmogenic cardiomyopathy (AC). 
PMID: 30796961

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
609040 
  PMID: 15489853   2/11
PMID:16549640   8/25

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
